packages/tradershub/src/hooks/usePlatformAssets.ts

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from 'react';
+import { useExchangeRates, useTradingAccountsList } from '@deriv/api';
+
+/**
+ * @description This hook is used to get the total demo and real balance of the user
+ * @param regulation  - Regulation of the user
+ */
+const usePlatformAssets = (regulation?: string) => {
+    const {
+        data: tradingAccount,
+        fiat_account: fiatAccount,
+        isSuccess: isTradingAccountSuccess,
+    } = useTradingAccountsList();
+    const { data, getExchangeRate, subscribe: multiSubscribe, unsubscribe } = useExchangeRates();
+
+    const isEURegulation = regulation === 'EU';
```

Comment:
Don't we have any constant for `EU | MF | CR`?

Reply:
we only have for EU. lemme change that

Reply:
done @shahzaib-deriv